### PR TITLE
add openwrt support

### PR DESCRIPTION
**Pull request by qdk0901**
Signed-off-by: qdk0901 <qdk0901@qq.com>

### CODE REVIEW

File: build/configs/toolchain_openwrt_mips.cmake [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2015 Samsung Electronics Co., Ltd.
```
Comment (ruben-ayrapetyan) [inline build/configs/toolchain_openwrt_mips.cmake]: Please, update the copyright year to 2016.

**Comment by ruben-ayrapetyan** [comment]
Looks good to me

**Comment by zherczeg** [comment]
LGTM

**Comment by ruben-ayrapetyan** [comment]
@qdk0901, could you, please, combine your commits into one?

Combining commits could be accomplished with the following command sequence:
1. `git remote add upstream_master https://github.com/Samsung/jerryscript.git`
2. `git fetch upstream_master`
3. `git rebase -i upstream_master/master`
4. edit the opened file, putting `fixup` instead of `pick` on second line; close the file
5. check that commits were actually combined and are rebased on current master using `git log` and `git diff upstream_master/master`
6. `git push --force origin branch_name`

**Comment by qdk0901** [comment]
@ruben-ayrapetyan thanks your instructions, it's combined now

**Comment by ruben-ayrapetyan** [comment]
@qdk0901, I'm sorry. I've missed that the `Signed-off-by` is in incorrect format.
I already started merge procedure, but precommit checker catched the issue.

Please, update it to `JerryScript-DCO-1.0-Signed-off-by: qdk0901 qdk0901@qq.com`.

You could perform this using `git commit --amend`, and then `git push --force origin branch_name`

**Comment by qdk0901** [comment]
@ruben-ayrapetyan it has been updated now.

**Comment by ruben-ayrapetyan** [comment]
@qdk0901, thank you!

The pull request is merged (columnIndex)

**Comment by ruben-ayrapetyan** [comment]
@qdk0901, thank you!

The pull request is merged (db6caf3c4830c352bc45a9178a43aba909574ba8)

**Pull request merged**